tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -24,3 +33,18 @@ Verify User Can Spawn A Notebook
     Wait For RHODS Dashboard To Load
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook With Arguments    image=s2i-minimal-notebook    size=Default
+
+Verify User Is Unable To Spawn Notebook
+    [Documentation]    Verifies User is unable to access notebooks in jupyterhub
+    Launch Dashboard    ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}  ${ODH_DASHBOARD_URL}  ${BROWSER.NAME}  ${BROWSER.OPTIONS}
+    Launch JupyterHub From RHODS Dashboard Link
+    Login To Jupyterhub  ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ${authorization_required} =  Is Service Account Authorization Required
+    Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+    User Is Not Allowed
```

Comment:
these lines could we replaced by 2 lines only:
```
Launch JupyterHub From RHODS Dashboard Link
Login Verify Access Level  ${TEST_USER.USERNAME}
    ...                                   ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}    none
```

Reply:
Updated.